ethereum/core/src/main/java/org/hyperledger/besu/ethereum/vm/operations/MulModOperation.java

```diff
@@ -33,12 +35,15 @@ public Gas cost(final MessageFrame frame) {
 
   @Override
   public void execute(final MessageFrame frame) {
-    final UInt256 value0 = frame.popStackItem().asUInt256();
-    final UInt256 value1 = frame.popStackItem().asUInt256();
-    final UInt256 value2 = frame.popStackItem().asUInt256();
-
-    final UInt256 result = value0.timesModulo(value1, value2);
+    final UInt256 value0 = UInt256.fromBytes(frame.popStackItem());
+    final UInt256 value1 = UInt256.fromBytes(frame.popStackItem());
+    final UInt256 value2 = UInt256.fromBytes(frame.popStackItem());
 
-    frame.pushStackItem(result.getBytes());
+    if (value2.isZero()) {
```

Comment:
This optimization is out of scope of this PR in my opinion. All those arithmetic operations changes you made could be done in a follow up PR.

Reply:
Actually... well the bytes library and UInt256 really don't like when you divide by zero.
Like it would throw pretty hard.
And the Int256 class was really not all that much used and causing me more heartache than anything else, so I inlined everything, and that's what it looks like now.

Reply:
Ok. Sounds good.